docs/upgrade.md

```diff
@@ -0,0 +1,20 @@
+# Upgrade
+
+This article walks you through how to upgrade your Coder server.
+
+## Upgrading Coder
+
+To upgrade your Coder server, simply re-run the `install.sh` script on the Coder
+server host:
+
+```console
+curl -L https://coder.com/install.sh | sh
+```
```

Comment:
We should probably direct people to reinstall using their original method of install. If they don't, they may end up with two Coder installations and that could be confusing.

Reply:
@ammario I split the upgrade path into two sections, one with folks who installed via `install.sh` and another for those who used our `docker-compose.yaml`.